librec/src/test/java/librec/util/GammaTest.java

```diff
@@ -0,0 +1,56 @@
+package librec.util;
+
+import librec.util.Gamma;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Method;
+
+public class GammaTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: Gamma */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 68 branch to line 69
+   */
+
+  @Test
+  public void digammaInputNegativeInfinityOutputNaN() {
+
+    // Arrange
+    final double x = Double.NEGATIVE_INFINITY;
+
+    // Act
+    final double retval = Gamma.digamma(x);
+
+    // Assert result
+    Assert.assertEquals(Double.NaN, retval, 0.0);
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 68 branch to line 68
+   *  - conditional line 68 branch to line 72
+   *  - conditional line 72 branch to line 73
+   */
+
+  @Test
+  public void digammaInputZeroOutputNegativeInfinity() {
```

Comment:
I also like this! Are there any digamma tests with inputs that are natural numbers?